Dockerfile

```diff
@@ -26,6 +24,7 @@ RUN mkdir /opt/apt-local-repository
 COPY docker/mysql/* /opt/apt-local-repository/
 RUN /opt/apt-local-repository/local-mysql-repos.sh
 
+RUN curl -fsSL https://deb.nodesource.com/setup_18.x | bash -
```

Comment:
We shouldn't run random bash scripts from the internet through bash if we can help it. Updating the nodesource gpg key and fixing the source list should be enough.

Reply:
Just following their own [instructions](https://github.com/nodesource/distributions?tab=readme-ov-file#installation-instructions) where they are strongly recommending we use this approach.

https://github.com/nodesource/distributions/wiki/Repository-Manual-Installation#-simplified-installation-using-scripts

Reply:
I'm also not sure I would call an officially supported setup script that is the recommended way to install the project that is shipped by the project itself a "random bash script from the internet".

Reply:
Other Mozilla projects are doing the same thing as well: https://github.com/mozilla/glean.js/pull/1807/files In case social credit means anything here.

Reply:
BTW this gets us on 18.19.